spec/features/questions/new_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+require 'pry'
```

Comment:
I think you can remove this require here.

Reply:
ok